pinax/api/jsonapi.py

```diff
@@ -43,7 +46,7 @@ def from_validation_error(cls, exc, resource_class):
                 errs.append(err)
         return cls(errors=errs)
 
-    def __init__(self, data=None, errors=None, links=False, included=None, meta=None, linkage=False):
+    def __init__(self, data=None, errors=None, links=False, included=None, meta={}, linkage=False):
```

Comment:
I see the reason. You need `self.meta` to be a dictionary in `get_serializable_data`. That makes sense. I say go for the first solution and in `serializable` don't check `if self.meta is not {}` but `if self.meta` (because `{} is not {}` is always `True`).